content/components/core/code/scss-functions.md

```diff
@@ -167,7 +167,9 @@ Add the outline to focus.
 
 ### AU-sronly
 
-Hide an element from the screen but not a screen reader.
+Hide an element from the screen but not a screen reader. This mixin should be used to provide additional information or context to a webpage or form that screenreader users would otherwise miss out on. Using this mixin the additional information will be picked up by a screenreader but will not be displayed in the browser. 
```

Comment:
```suggestion
Hide an element from the screen but not a screen reader. This mixin should be used to provide additional information or context to a webpage or form that the screen reader users would otherwise miss out on. With this mixin, the additional information will be picked up by a screen reader but will _not_ be displayed in the browser. 
```